src/Components/Votes.jsx

```diff
@@ -1,9 +1,23 @@
+import { useEffect, useState } from 'react';
 import './Component-Style/Votes.css'
+import DownvoteArrow from './DownvoteArrow';
+import UpvoteArrow from './UpvoteArrow';
 
-const Votes = ({votes}) => {
-    return <section>
-        Votes: {votes}
+const Votes = ({article_id, votes}) => {
+    const [displayVotes, setDisplayVotes] = useState(0);
+
+    useEffect(()=> {
+        setDisplayVotes(votes);
+    },[votes])
+
+    return <section className='votes'>
+        <div>
+            Votes: {displayVotes}
+        </div>
```

Comment:
`<div>` is a great tag for grouping and styling but there are better tags like `<p>` for text!